src/server.js

```diff
@@ -46,6 +47,9 @@ module.exports = function() {
 		.use("/storage/", express.static(Helper.getStoragePath(), {
 			redirect: false,
 			maxAge: 86400 * 1000,
+		})).use("/uploads/", express.static(Helper.getFileUploadPath(), {
```

Comment:
We'll want to figure something to make most files have `plain/text` content type, because we don't want to make js/css/etc files arrive with their respective content types.

Reply:
Any thoughts on sending `Content-Disposition: inline` as well to encourage browsers to display instead of prompting to save? (If The Lounge saves the original filename anywhere, that could be passed along in this header too, I think.)